lib/secure.js

```diff
@@ -100,8 +100,17 @@ module.exports = function(getKey, options) {
                 return false;
             }
 
-            if (expectedClaim.value && claim !== expectedClaim.value) {
-                return false;
+            if (Array.isArray(claim)) {
```

Comment:
As we're using `jose` here anyway, the next step could be extracting claims like `aud`, `iss`, `sub` from the `expectedClaims` and passing them to the `jwtVerify` method. That's a nice-to-have.